index.html

```diff
@@ -304,5 +324,114 @@ <h3 class="card__title">Skills</h3>
+              required
+            />
+            <input
+              type="email"
+              placeholder="Email address"
+              id="contact-form__email"
+              name="email"
+              required
+            />
+            <textarea
+              name="message"
+              id="contact-form__message"
+              cols="30"
+              rows="5"
+              placeholder="Enter text here"
+              maxlength="500"
+              required
+            ></textarea>
+            <button class="btn btn--primary">Get in touch</button>
+          </form>
+        </div>
+      </div>
+    </section>
+    <footer id="footer">
+      <div class="wrapper">
+        <div class="footer__inner">
+          <ul class="headline-social-icons">
+            <li class="headline-social-icon">
+              <a
+                href="https://twitter.com/shahadat3669"
+                class="headline-social-icon-link"
+              >
+                <img
+                  src="./dist/images/social-icons/twitter.svg"
+                  alt="twitter icon"
+                  class="headline-social-icon-img"
+                />
+              </a>
+            </li>
+            <li class="headline-social-icon">
+              <a
+                href="https://linkedin.com/in/shahadat3669"
+                class="headline-social-icon-link"
+              >
+                <img
+                  src="./dist/images/social-icons/Linkedin.svg"
+                  alt="linkedin icon"
+                  class="headline-social-icon-img"
+                />
+              </a>
+            </li>
+            <li class="headline-social-icon">
+              <a
+                href="https://medium.com/@shahadat3669"
+                class="headline-social-icon-link"
+              >
+                <img
+                  src="./dist/images/social-icons/medium.svg"
+                  alt="medium icon"
+                  class="headline-social-icon-img"
+                />
+              </a>
+            </li>
+            <li class="headline-social-icon">
+              <a
+                href="https://github.com/shahadat3669"
+                class="headline-social-icon-link"
+              >
+                <img
+                  src="./dist/images/social-icons/github.svg"
+                  alt="github icon"
+                  class="headline-social-icon-img"
+                />
+              </a>
+            </li>
+            <li class="headline-social-icon">
+              <a href="https://angel.co/" class="headline-social-icon-link">
+                <img
+                  src="./dist/images/social-icons/angellist.svg"
+                  alt="angellist icon"
+                  class="headline-social-icon-img"
+                />
+              </a>
+            </li>
+          </ul>
+          <a class="footer__email" href="mailto:shahadat3669@gmail.com">
+            shahadat3669@gmail.com
```

Comment:
[OPTIONAL]Kindly add the bottom border that is on this page, to match the figma design. It's important to showcase that you can meet design requirements when working with a wireframe or figma design.